ecl/hql/hqlrepository.cpp

```diff
@@ -676,6 +676,8 @@ unsigned __int64 EclRepositoryManager::getStatistic(StatisticKind kind) const
     {
     case StTimeElapsed:
         return cycle_to_nanosec(gitDownloadCycles);
+    case StTimeBlocked:
+        return cycle_to_nanosec(gitDownloadBlockedCycles);
```

Comment:
I am not sure the context of when this getStatistic is called, but should these counters really atomic's ?

Reply:
no, they are only called from a single thread.